views/js/qtiCommonRenderer/helpers/instructions/instructionManager.js

```diff
@@ -182,7 +182,7 @@ define([
             }
 
             if(!minInstructionSet && min > 0 && (choiceCount === false || min < choiceCount)){
-                msg = (min <= 1) ? __('You must at least %d choice', min) : __('You must select at least %d choices', min);
+                msg = __('You must select at least %d choices', min);
```

Comment:
The message should be depending on the number of min choices
`msg = (min <= 1) ? __('You must select at least %d choice', min) : __('You must select at least %d choices', min);` would be correct

Reply:
of course. this is my mistake.